WDP190404-12 -->Add RWD topbar and banner

HRDagmara opened the pull request.

lpi80 commented:
Do rozdzielczości 523x329 wszystko wygląda fajnie.
Jednak wydaje mi się, że coś byłoby trzeba poprawić poniżej tej wielkości (elementy zaczynają dziwnie się zachowywać)

Varenthein commented:
Add some decription please

HRDagmara commented:
Opis: 

Strona sypie się w trybach responsive. Klient nie ma designów RWD, więc trzeba zrobić na własną rękę. 

Ten task dotyczy górnego paska (z walutą, językiem, etc.) oraz bannera na którym jest logo i inne elementy. 

Na tabletach wygląda ok, ale na mobilkach trzeba poprawić: 

    top bar
        dropdowny po lewej mają pozostać bez zmian
        login i register mają być bez tekstów (tylko ikony)
        w razie potrzeby można zmniejszyć marginy/paddingi wszystkich elementów, ale tylko w trybach w których jest to potrzebne
        ta ikona menu po prawej na topbarze ma zostać, zarówno na mobilkach jak i desktopie, Klient później zdecyduje co chce tam mieć i czy zmieni się ikona
    banner
        po lewej ma być logo
        po prawej koszyk (wyrównany do góry logo, czyli trochę wyżej niż jest)
        po koszykiem ma być telefon (ikona i numer)
        numer telefonu można zmniejszyć do 14px

Rozwiązanie: 
--header.scss

    dodanie RWD do topbar i banner

HRDagmara commented:
Dodałam zmiany na szerokościach poniżej 523px w _header.scss

HRDagmara commented:
> 
> 
> It's not a disaster and it looks acceptable, but I think we can improve a few things.
> 
> ![image](https://user-images.githubusercontent.com/13353257/56078524-87e1dc00-5de9-11e9-9fac-e2307894028b.png)
> 
>     1. Why there is so much margin at the right? I'm talking about the margin at the right of the cart and telephone number.
> 
>     2. We have a lot of space between English and Login. Why don't we use that? It would look better if we do that instead of setting help to next line.

I modified RWD for mobile.  Now, I think it looks better :)

Varenthein commented:
It's almost perfect, but take we have to take care of small mobiles. That's how everything looks on 460px screen.

![image](https://user-images.githubusercontent.com/13353257/56225852-68a9af80-6072-11e9-80cd-414389807567.png)

It will be easy to fix :)

1. Make phone number smaller (change font-size)
2. Split two columns in topbar in a different way (for instance col-7, col-5)

That's all :)

HRDagmara commented:
> 
> 
> It's almost perfect, but take we have to take care of small mobiles. That's how everything looks on 460px screen.
> 
> ![image](https://user-images.githubusercontent.com/13353257/56225852-68a9af80-6072-11e9-80cd-414389807567.png)
> 
> It will be easy to fix :)
> 
>     1. Make phone number smaller (change font-size)
> 
>     2. Split two columns in topbar in a different way (for instance col-7, col-5)
> 
> 
> That's all :)

I modified font-size and add col :)

agmatyja commented:
Smaller letters and/or no spaces between parts of phone-number and/or modyfied margins could improve mobile version (if all those options are acceptable ;-) )

HRDagmara commented:
> 
> 
> Smaller letters and/or no spaces between parts of phone-number and/or modyfied margins could improve mobile version (if all those options are acceptable ;-) )

I modified RWD .I think to 320px I don't have 'break effect'.

lpi80 commented:
Przy najmniejszych rozdzielczościach coś mi źle wyświetla:
![image](https://user-images.githubusercontent.com/44462376/56412791-9b77c180-6285-11e9-8808-ad3f78da8f60.png)

a tutaj jeszcze dobrze:
![image](https://user-images.githubusercontent.com/44462376/56412813-af232800-6285-11e9-955e-5cf66fb81a3b.png)

Płynność przechodzenia podczas zmieniania wielkości okna fajnie działa
